packages/block-library/src/gallery/editor.scss

```diff
@@ -38,13 +38,25 @@ figure.wp-block-gallery {
 	}
 
 	figure.is-selected {
-		box-shadow: 0 0 0 $border-width $white, 0 0 0 3px var(--wp-admin-theme-color);
+
 		border-radius: $radius-block-ui;
```

Comment:
🎉 ! I noticed this during our 🍐 'ing session @karmatosed . The corners of the image seem to be peaking out from the focus ring. I think that would be resolved if we add this back in

Reply:
Unfortunately when I add that back in the overlap returns. There is the option of doing 'box-shadow: 0 0 0 $border-width $white;' but it then gives a slightly weird overlap. I'm very open to any ideas here as would love to get this smooth visually.